README.md

```diff
@@ -378,7 +378,40 @@ server {
         proxy_set_header Host $host;
 
         client_max_body_size 50M;
-    
+    }
+}
+
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    access_log /var/log/nginx/ess.log main;
+    error_log /var/log/nginx/ess.errors;
+
+    ssl_certificate /etc/nginx/certs/certificate.full;
+    ssl_certificate_key /etc/nginx/certs/certificate.key;
+
+    #TLSv1.2 is required for iOS support for now
+    ssl_protocols TLSv1.2 TLSv1.3;
+    ssl_dhparam /etc/nginx/dhparam.pem;
+    ssl_session_cache shared:le_nginx_SSL:10m;
+    ssl_session_timeout 1440m;
+    ssl_session_tickets off;
+    ssl_buffer_size 4k;
+    ssl_stapling on;
+    ssl_stapling_verify on;
+    add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload' always;
+    ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:DHE-RSA-CHACHA20-POLY1305;
+    ssl_prefer_server_ciphers on;
+
+    server_name mrtc.example.com;
+
+    location / {
+        proxy_pass http://127.0.0.1:8080;
+        proxy_set_header X-Forwarded-For $remote_addr;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host $host;
+
         proxy_http_version 1.1;
```

Comment:
These lines should only be on the SFU path, where as the lines above should be for all paths